server/views/pages/locationDetails.njk

```diff
@@ -96,8 +96,8 @@
     <h2 class="govuk-!-font-size-36">Previous locations</h2>
     <div data-qa="previous-locations">
       {% for agency in locationDetailsGroupedByAgency %}
-        {% set agencyDatePeriod = "from " + agency.fromDateString + " to " + agency.toDateString %}
-        {% if agency.isValidAgency %}
```

Comment:
Removed need for explicit boolean that just checked if the agencyName had a value.